### PR TITLE
Use new markup parser

### PR DESCRIPTION
**Pull request by djmetzle**
In the previous PR: #68 we built a new, simpler parser implementation. Lets use it.

This replaces `markup_to_json` with the "new" implementation everywhere, and drops the old one.

I've been testing, and we see the same test behavior with either version.
qa_req 0

CC @andyg0808

### CODE REVIEW

**Comment by klmork** [comment]
CR 🌸

**Review by klmork — DISMISSED** [review]

File: src/markup_to_json.test.js [hunk 1]
```diff
@@ -14,30 +14,13 @@ const TestCases = [
...
-      `./test/${testcase}.source.jpg`,
-      "./literally.anywhere"
-    );
-    expect(testJSON).toEqual(expectedJSON);
-  });
-});
-
-const newConvertMarkupToJSON = require("./new_markup_to_json");
-
 describe("New Markup parser `newConvertMarkupToJSON`", () => {
```
Comment (klmork) [inline src/markup_to_json.test.js:17]: Oh, might want to change text here?

Comment (klmork) [reply src/markup_to_json.test.js:17]: newConvertMarkupToJSON to convertMarkupToJSON

Comment (djmetzle) [reply src/markup_to_json.test.js:17]: Oh good catch.

Comment (djmetzle) [reply src/markup_to_json.test.js]: Fixed in https://github.com/iFixit/node-markup/pull/69/commits/fe7e18fbbf47e5f271844d97c01d159413fa314e ?

**Comment by klmork** [comment]
CR 🌸

**Review by klmork — APPROVED** [review]

**Pull request merged**